src/components/FeedbackWidget.tsx

```diff
@@ -220,82 +124,131 @@ const FeedbackWidget: React.FC<IProps> = ({ className }) => {
   useKeyPress(`Escape`, handleClose)
 
   if (!location.includes("/en/")) return null
-
+  const closeButtonSize = "24px"
   return (
     <>
       <FixedDot onClick={handleOpen} bottomOffset={bottomOffset} id="dot">
-        <StyledFeedbackGlyph />
+        <Icon as={FeedbackGlyph} color="white" h="32px" w="26px" />
       </FixedDot>
       {isOpen && (
-        <ModalBackground>
+        <Box
+          display="block"
+          position="fixed"
+          inset={0}
+          bgColor="blackAlpha.400"
+          zIndex={1001} /* Above the nav bar */
+        >
           <FocusTrap
             focusTrapOptions={{
               fallbackFocus: `#dot`,
             }}
           >
-            <Container
-              bottomOffset={bottomOffset}
-              ref={containerRef}
-              className={className}
+            <Flex
               id="modal"
+              ref={containerRef}
```

Comment:
I think we should aim to use the Chakra Popover in this component. https://chakra-ui.com/docs/components/popover we will have better a11y and will abstract us for a bunch of things we are doing here (close icon, modal styles, focus trap, etc).

Not saying this should be in the scope of this PR but I'll create a new issue to tackle that after we merge this.